spec/api-app-spec.js

```diff
@@ -511,13 +510,7 @@ describe('app module', () => {
       expect(app.getLoginItemSettings().openAsHidden).to.be.false()
     })
 
-    it('allows you to pass a custom executable and arguments', function () {
-      if (process.platform !== 'win32') {
-        // FIXME(alexeykuzmin): Skip the test.
-        // this.skip()
-        return
-      }
-
+    platformIt('allows you to pass a custom executable and arguments', ['win32'], function () {
```

Comment:
Looks useful.
We also need `xplatformIt` and `xplatformDescribe` to match `xit` and `xdescribe` to temporary disable tests.